prow/jobs/kyma/tools/failery/failery.yaml

```diff
@@ -49,6 +49,16 @@ presubmits: # runs on PRs
       labels:
         <<: *job_labels_template
         preset-build-release: "true"
+    - name: pre-rel08-kyma-tools-failery
```

Comment:
This doesn't have `always_run: true`. Could you make sure if this was intended or a mistake?

Reply:
forgotten
and fixed in next push